nodejs/week2/homework/meal-sharing/src/backend/api/meals-router.js

```diff
@@ -0,0 +1,83 @@
+const express = require("express");
+const router = express.Router();
+
+const meals = require("./../data/meals.json");
+
+router.get("/", async (req, res) => {
+  if(Object.keys(req.query).length === 0 ){
+  res.send(meals);
+} 
+else if
+(req.query.hasOwnProperty("id")){
```

Comment:
Not exactly! The hw wants you to create a new route like this: 
```/api/meals/{id}```
So in the new route instead of having the id in the request's query it should be in its params -> read carefully here: https://github.com/HackYourFuture-CPH/node.js/tree/main/week2/homework#routes